doc/user/load_save.md

```diff
@@ -1,4 +1,4 @@
-<object data="../img/pipeline-top-1.svg" type="image/svg+xml" id="pipeline-top">
+<object data="../img/pipeline-top-1.svg" type="image/svg+xml" id="pipeline-top">loadsave
```

Comment:
Accidental checkin?  (Maybe there is some reason it's needed that I don't know?)